_i18n/en/documentation/bugs-first-aid/refresh-download-errors.md

```diff
@@ -0,0 +1,12 @@
+Do you get a notification saying that downloads completed with errors? Then AntennaPod could not retrieve the feed with episode information or an audio files from the website of the podcast hoster.
+
+Issues that only occur with one or a few specific podcasts are often related to those podcasts. To confirm if this is the case, you can follow these steps:
+1. tap on the notification to open the Download Log
+2. tap on the specific notification with the error
+3. long-press on the URL that is displayed, and copy it or open it in your browser
+
+If the URL doesn't work in the browser either, this means that it is not a problem with the app. If you get the error message consistently then the podcast has likely been moved, taken off-line or not properly configured.
+
+Does the error message in the Download log say 'Failed to connect to [url]'? Then this means that something is interfering with your network. If you have an ad-blocker, VPN, captive portal, proxy, privacy guard, special DNS service, or parental control set up on your phone, you can try turning these off.
```

Comment:
I think all error messages caused by apps messing with the network connection already say so in the error message. A generic error could be caused by something else. Not sure.

Reply:
Right. Well, let's keep it then. Can always adjust it if we get more info in future. Wouldn't know what/how to write differently.